src/sagas/SignUp.test.ts

```diff
@@ -0,0 +1,110 @@
+import { delay } from 'redux-saga'
+import { takeLatest, call, put } from 'redux-saga/effects'
+import { describe } from 'riteway'
+
+import { Actions } from '../actions/index'
+import { SignUpSaga, SignUp, signUpFrost } from './SignUp'
+
+describe('SignUp Saga', async (should: any) => {
+  const { assert } = should()
+
+  const iterator = SignUpSaga()()
+  assert({
+    given: 'sign up action',
+    should: 'handle sign up request',
+    actual: iterator.next().value,
+    expected: takeLatest(Actions.SignUp.SIGN_UP, SignUp),
+  })
+})
+
+describe('SignUp() Success', async (should: any) => {
+  const { assert } = should()
+  const email = 'test@email.com'
+  const password = 'test'
+  const token = 'test-token'
+  const iterator = SignUp(Actions.SignUp.onSignUp({ email, password }))
+  assert({
```

Comment:
can you add an empty line before `assert`?

Reply:
Done!